kie-server-parent/kie-server-controller/kie-server-controller-websocket-common/src/main/java/org/kie/server/controller/websocket/common/config/WebSocketClientConfiguration.java

```diff
@@ -61,6 +72,21 @@ public Builder setMaxSessionIdleTimeout(final Long maxSessionIdleTimeout) {
             return this;
         }
 
+        public Builder setAsyncSendTimeout(final Long asyncSendTimeout) {
```

Comment:
where do we use async send?

Reply:
https://github.com/cristianonicolai/droolsjbpm-integration/commit/aa7ec7002d716f2c051e6b6754dc1aab72a13a66#diff-ba458fffe022fec7358b028ed06b5d03R52